src/schemas/json/swcrc.json

```
@@ -535,6 +540,11 @@
               "type": "string",
               "$comment": "Requires swc@1.2.62",
               "description": "Should reflect \"baseUrl\" property in project's tsconfig.json . See https://www.typescriptlang.org/docs/handbook/module-resolution.html#base-url"
+            },
+            "minify": {
+              "type": "object",
+              "$comment": "Requires swc@1.2.67",
+              "description": "See https://swc.rs/docs/configuration/minification"
             }
```

```suggestion
            "minify": {
              "title": "minification options",
              "type": "object",
              "$comment": "Requires swc@1.2.67",
              "description": "Minification options\nhttps://swc.rs/docs/configuration/minification"
            }
```

Almost the same issue as above.
Optional fix: `title` key can be added to enhance VS Code editing experience.
Optional fix: all minification keys can be added.